### PR TITLE
Lis cws facility

### PR DESCRIPTION
**Pull request by Nasir2712**
#### Which User story does this relate to?
- https://osi-cwds.atlassian.net/browse/CAL1-3

#### Brief feature description
- Check to see if facility source is loaded in both grid view and list view if it exists.

#### Does this PR block/effect other stories?
- no

#### Do any commands needs to be run in order to run the project with this PR?
- no

#### Code coverage percent diff as a result of this PR?
-

### CODE REVIEW

File: app/javascript/search/search_grid.js [hunk 1]
```diff
@@ -31,14 +31,17 @@ export default class SearchGrid extends React.Component {
               <GridInnerLayout
                 title={'Facility Type'}
                 value={checkForNA(result.type)} />
+              <GridInnerLayout
+                title={'Facility Source'}
```
Comment (ratneshraval) [inline app/javascript/search/search_grid.js]: strings with `{ }` :(

File: app/javascript/search/search_list.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ export default class SearchList extends React.Component {
             <tr>
               <th>Facility Name </th>
               <th>Facility license/ Approval #</th>
-              <th>Facility Type</th>
+              <th>Facility Type/ Facility Source</th>
```
Comment (ratneshraval) [inline app/javascript/search/search_list.js]: spacing issue here?

Comment (Nasir2712) [reply app/javascript/search/search_list.js]: no, so if you see facility license/ Approval leon told me to do keep it exactly like that.

Comment (ratneshraval) [reply app/javascript/search/search_list.js]: so not `Facility license / Approval`?

Comment (Nasir2712) [reply app/javascript/search/search_list.js]: Well i don't know about that. He told me to replicate how the Facility Type/ Approval was done so i just went with it. If you want, I can add a space in both those fields.

File: app/javascript/search/search_grid.js (hunk 1 above)
Comment (Nasir2712) [reply app/javascript/search/search_grid.js]: done

File: app/javascript/search/search_list.js (hunk 2 above)
Comment (ratneshraval) [reply app/javascript/search/search_list.js]: I think it makes sense to have even spacing. I'll approve, can you fix the spacing please.

**Pull request merged**